hacking/aws_config/testing_policies/security-policy.json

```diff
@@ -132,26 +116,16 @@
             "Resource": "*"
         },
         {
-            "Sid": "AllowAccessToSpecifiedIAMResources",
+            "Sid": "AllowAccessToServerCertificates",
             "Effect": "Allow",
             "Action": [
-                "iam:CreateRole",
-                "iam:DeleteRole",
-                "iam:GetRole",
-                "iam:PassRole",
-                "iam:UpdateAssumeRolePolicy"
```

Comment:
Maybe not likely, but it seems possible if a test uses a static role that isn't cleaned up at the end of a test run and it, is modified, then rerunning the tests would try to use this permission. It doesn't look like it's duplicated anywhere in the policies.